config/Dockerfiles/tests.d/openstack/02_os-server-template

```diff
@@ -1,9 +1,8 @@
 #!/bin/bash -xe
 
 # Verify os-server provisioning using inline template data
-# distros.exclude: none
-# providers.include: openstack
-# providers.exclude: none
+# distros.exclude: centos7 fedora29 fedora30
+# providers.include: none
```

Comment:
Why are we disabling this test?

Reply:
templating functionality is being tested in dummy provider and unit tests i believe there is no reason for us to test it again since we already provision openstack instances.